packages/file_selector/file_selector_linux/lib/file_selector_linux.dart

```diff
@@ -16,6 +16,7 @@ const String _typeGroupMimeTypesKey = 'mimeTypes';
 const String _openFileMethod = 'openFile';
 const String _getSavePathMethod = 'getSavePath';
 const String _getDirectoryPathMethod = 'getDirectoryPath';
+const String _getDirectoriesPathsMethod = 'getDirectoriesPaths';
```

Comment:
This should not use a new method. Please follow the same structure as `openFile`.